src/apps/frontend/types/auth.ts

```diff
@@ -9,3 +9,12 @@ export default class AccessToken {
     this.token = json.token as string;
   }
 }
+
+export enum KeyboardKeys {
```

Comment:
React TS should have this already

Reply:
I didn't found. I think there is no built-in type or enum for `Backspace` keyboard key. please can you help me?